src/config/chis-tg/config.json

```diff
@@ -0,0 +1,189 @@
+
+{
+  "domains": [
+    {
+      "friendly": "MoH Togo DEV",
+      "domain": "moh-togo.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH Togo UAT",
```

Comment:
did togo go live? is there a production url?

Reply:
@paulpascal  will know more accurately, but I believe [we did the work](https://github.com/medic/medic-infrastructure/issues/782) resulting in `moh-togo.app.medicmobile.org`